nextflow.config

```diff
@@ -66,11 +66,14 @@ params {
     min_read_counts                     = null
     flye_mode                           = "--nano-corr"
 
-
     // Screen pathogens
     perform_screen_pathogens            = false
     accession2taxid                     = null
 
+    // Consensus calling
+    longread_consensus_tool             = 'medaka'
```

Comment:
Why have the tests as default samtools but here is medaka?

Reply:
Shall the `min_read_counts` be added too?

Reply:
> Why have the tests as default samtools but here is medaka?

Because the test data is too small to run `medaka`, I am testing it on the full validation dataset. In practice, `medaka` performs better than `samtools`, so I think we can keep it as it is.